content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,120 @@
+
+By marking Secrets and ConfigMaps as immutable you can be sure your application configuration won't change. If you want to make changes you'll need to create a new, uniquly named Secret or ConfigMap and deploy a new pod to consume that resource. Immutable resources also have scaling benefits because controllers do not need to poll the API server to watch for changes.
+This feature is now graduating to stable in Kubernetes 1.21.
+
+### Graceful Node Shutdown
+Graceful Node shutdown is graduating to beta with this release (and will now be available to a much larger group of users!). This is a hugely beneficial feature that allows Kubelet to be aware of Node shutdown, and gracefully terminate Pods that are scheduled to that Node.
+
+Currently, when a node shuts down, Pods do not follow the expected termination lifecycle and are not shutdown gracefully. This can introduce problems with a lot of different workloads. Going forward, Kubelet will be able to propagate the shutdown signal to Pods ensuring they can terminate as gracefully as possible.
+
+### PV Health Monitor
+Physical Volumes (PV) are commonly used in applications to get local, file-based storage. They can be used in many different ways and help users migrate applications without needing to re-write storage backends.
+
+Kubernetes 1.21 has a new alpha feature which allows PVs to be monitored for health of the volume and marked accordingly if the volume becomes unhealthy. Workloads will be able to react to the health state to protect data from being written or read from a volume that is unhealthy.
+
+### Reducing Kubernetes Build Maintenance
+Previously Kubernetes has maintained multiple build systems. This has often been a source of friction and complexity for new and current contributors.
+
+Over the last release cycle, a lot of work has been put in to simplify the build process, and standardize on the native Golang build tools. This should empower broader commmunity maintenance, and lower the barier to entry for new contributors.
+
+## Major Changes
+
+### PodSecurityPolicy Deprecation
+As of Kubernetes 1.21 PodSecurityPolicies in their current form will be deprecated. Even though the feature has been deprecated, it will continue to be available and function as before until Kubernetes 1.25. Going forward, there is a proposal in place for a simplified feature that will fill the basic use cases of PodSecurityPolicies.
+
+SIG security has prepared a dedicated blog post with more details about the PodSecurityPolicy feature deprecation. You can read that [here.]()
+
+## Other Updates
+
+### Graduated to Stable
+
+* [EndpointSlice](https://github.com/kubernetes/enhancements/issues/752)
+* [Add sysctl support](https://github.com/kubernetes/enhancements/issues/34)
+* [PodDisruptionBudgets](https://github.com/kubernetes/enhancements/issues/85)
+
+### Notable Feature Updates
+
+* [External client-go credential providers](https://github.com/kubernetes/enhancements/issues/541)
```

Comment:
What's the update? Is it a graduation to stable?

Reply:
I've marked it as alpha in 1.21. PTAL & lmk  if that's fine :)